src/webgpu/shader/execution/builtin/fract.spec.ts

```diff
@@ -49,15 +49,17 @@ T is f32 or vecN<f32> fract(e: T ) -> T Returns the fractional bits of e (e.g. e
       // Min and Max f32
       { input: f32Bits(kBit.f32.positive.min), expected: f32Bits(kBit.f32.positive.min) },
       { input: f32Bits(kBit.f32.positive.max), expected: f32(0) },
-      { input: f32Bits(kBit.f32.negative.max), expected: f32Bits(0x3f7fffff) },
+      { input: f32Bits(kBit.f32.negative.max), expected: anyOf(f32Bits(0x3f7fffff), f32(1)) },
```

Comment:
Here and below running the tests on NVIDIA vs SwiftShader they disagree on 1 vs very close to 1. I think what is occurring is that if the calculation is internally done in f32, i.e. rounding/flushing each step, you end up with one value, but all of the math is done in a higher precision, and then converted at the end, you end up with a different value.

I am not sure what we want to about this, or if this is fine, so I encoded the most permissive form of the check for now.